test/validate-admin-test.js

```diff
@@ -0,0 +1,30 @@
+const buster = require('buster')
+const sinon = require('sinon')
+const validateAdmin = require('../lib/validate-admin')
+
+buster.testCase('validateAdmin', {
+  setUp: function () {
+    this.res = { send: this.stub(), json: this.stub(), status: this.stub() }
+    this.res.status.returns(this.res)
+    this.next = this.stub()
+  },
+
+  'should fail when user was not an admin': function () {
+    const user = { name: '', password: '', isAdmin: false }
+    validateAdmin({ decoded: user }, this.res, this.next)
+    buster.assert.calledWith(this.res.json, sinon.match.has('success', false))
+    buster.assert.calledWith(this.res.json, sinon.match.has('message'))
+  },
+
+  'should set response code to forbidden 403 when user was not an admin': function () {
+    const user = { name: '', password: '', isAdmin: false }
+    validateAdmin({ decoded: user }, this.res, this.next)
+    buster.assert.calledWith(this.res.status, 403)
```

Comment:
Für Unit Tests, finde ich, sollte man eine Sache pro TestCase testen. Das ist sicherlich ein bisschen Geschmackssache, ob das noch das einer oder zwei Tests sein sollten. Aber auf jeden Fall sollte die Anzahl der Test-Zeilen nicht das ausschlaggebende sein. 

Für diese Datei speziell: ist noch nicht fertig und auch noch nicht integriert (siehe TODO Kommentar in `api.js`. Da muss noch ein bisschen was überarbeitet werden (ich bin dran).